modules/vcflib/vcfbreakmulti/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "vcflib_vcfbreakmulti"
+description: write your description here
+keywords:
+  - vcflib
+  - vcflib/vcfbreakmulti
+tools:
+  - "vcflib":
+      description: "Command-line tools for manipulating VCF files"
+      homepage: https://github.com/vcflib/vcflib
+      documentation: https://github.com/vcflib/vcflib#USAGE
+      doi: "https://doi.org/10.1101/2021.05.21.445151"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: Compressed VCF file
+      pattern: "*.vcf.gz"
```

Comment:
The tool could take uncompressed also right?

Reply:
Yes, solved :)